lib/rouge/guessers/disambiguation.rb

```diff
@@ -90,8 +90,13 @@ def match?(filename)
       disambiguate '*.m' do
         next ObjectiveC if matches?(/@(end|implementation|protocol|property)\b/)
         next ObjectiveC if contains?('@"')
-
-        next Mathematica if contains?('(*')
+        # Opening brace at end of line
+        next ObjectiveC if matches?(/\s*{\s*$/)
```

Comment:
Actually now that I've mentioned it, this might trigger ObjC for Mason files, which will often also have trailing open braces.

Reply:
Good catch. I'll create test cases for that and make any needed adjustments